cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,631 @@
+    search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+    clear_filters = Text(".//a[text()='Clear All Filters']")
+    # 5.9 field
+    migr_dropdown = MigrationDropdown(text="Not Started Plans")
+
+    @property
+    def in_migration_plan(self):
+        return (
+            len(self.browser.elements('.//div[contains(@class,"card-pf")]')) > 0
+            and len(self.browser.elements(PFIcon.icons.WARNING)) < 1
+        )
+
+    @property
+    def is_displayed(self):
+        return self.in_migration_plan
+
+
+class NotStartedPlansView(MigrationPlanView):
+    TITLE = "Not Started"
+    plans_not_started_list = MigrationPlansList("plans-not-started-list")
+
+    @property
+    def is_displayed(self):
+        def in_migration():
+            return self.in_migration_plan and self.plans_not_started_list.is_displayed
+
+        in_view = VersionPicker(
+            {
+                Version.lowest(): in_migration,
+                "5.10": in_migration and self.TITLE in self.title.read(),
+            }
+        ).pick()
+        return in_view
+
+
+class InProgressPlansView(MigrationPlanView):
+    TITLE = "In Progress"
+    # TODO - fix this once dev fixes the locator
+    progress_card = MigrationProgressBar(locator=".//div[3]/div/div[3]/div[3]/div/div")
+
+    @property
+    def is_displayed(self):
+        def in_migration():
+            return self.in_migration_plan and self.progress_card.is_displayed
+
+        in_view = VersionPicker(
+            {
+                Version.lowest(): in_migration,
+                "5.10": in_migration and self.TITLE in self.title.read(),
+            }
+        ).pick()
+        return in_view
+
+
+class CompletedPlansView(MigrationPlanView):
+    TITLE = "Completed"
+    plans_completed_list = MigrationPlansList("plans-complete-list")
+
+    @property
+    def is_displayed(self):
+        def in_migration():
+            return self.in_migration_plan and self.plans_completed_list.is_displayed
+
+        in_view = VersionPicker(
+            {
+                Version.lowest(): in_migration,
+                "5.10": in_migration and self.TITLE in self.title.read(),
+            }
+        ).pick()
+        return in_view
+
+
+class ArchivedPlansView(MigrationPlanView):
+    TITLE = "Archived"
+    migration_plans_list = MigrationPlansList("plans-archived-list")
+
+    @property
+    def is_displayed(self):
+        def in_migration():
+            return self.in_migration_plan and self.migration_plans_list.is_displayed
+
+        in_view = VersionPicker(
+            {
+                Version.lowest(): in_migration,
+                "5.10": in_migration and self.TITLE in self.title.read(),
+            }
+        ).pick()
+        return in_view
+
+
+class AddMigrationPlanView(View):
+    title = Text(locator='.//h4[contains(@class,"modal-title")]')
+    name = TextInput(name="name")
+    description = TextInput(name="description")
```

Comment:
I don't believe `name` or `description` are needed here since they are defined on the nested `general` class

Reply:
Removed